test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/SharedArchiveFileOption.java

```diff
@@ -228,6 +228,30 @@ private static void doTest(String baseArchiveName, String topArchiveName) throws
                   "-Xlog:cds",
                   "-cp", appJar, mainClass)
                  .assertAbnormalExit("Cannot use the following option when dumping the shared archive: --patch-module");
+
+            // following two tests:
+            //   -Xshare:auto -XX:SharedArchiveFile=top.jsa, but base does not exist.
+            if (!isUseSharedSpacesDisabled()) {
+                new File(baseArchiveName).delete();
+                testcase("Archive not loaded -XX:+AutoCreateSharedArchive -XX:SharedArchiveFile=" + topArchiveName);
+                run(topArchiveName,
+                    "-Xshare:auto",
+                    "-XX:+AutoCreateSharedArchive",
+                    "-cp",
+                    appJar, mainClass)
+                    .assertNormalExit(output -> {
+                        output.shouldContain("warning: -XX:+AutoCreateSharedArchive is unsupported when base CDS archive is not loaded");
```

Comment:
I think the above could be:
    output.shouldContain("warning: " + ERROR);

Reply:
In this case, ERROR is equal to `-XX:ArchiveClassesAtExit is....` not `-XX:+AutoCreateSharedArchive...`.

Reply:
Oh, I see. Looks good then.